mass2/core/misc.py

```diff
@@ -11,67 +15,74 @@
 
 
 def show(fig: plt.Figure | None = None) -> mo.Html:
+    """Create a Marimo interactive view of the given Matplotlib figure (or the current figure if None)."""
     if fig is None:
         fig = plt.gcf()
     return mo.mpl.interactive(fig)
 
 
 def pickle_object(obj: Any, filename: str) -> None:
+    """Pickle the given object to the given filename using dill."""
```

Comment:
add "Mass2 receipes are compatible with dill, but not comatible with the std library pickle functions."